### ISSUE TITLE
Dropout(0.9)

### ISSUE THREAD

**Issue by ZFTurbo** [bonlime/keras-deeplab-v3-plus#4]
I believe it should be:
Dropout(0.1)

Orig TF repo:
concat_logits = slim.dropout(
            concat_logits,
            keep_prob=0.9,
            is_training=is_training,
scope=_CONCAT_PROJECTION_SCOPE + '_dropout')

Keep prob means that we keep 0.9 of probabilities.

**Comment by bonlime** [comment]
You are right. Fixed it
